logs/logs.html

```diff
@@ -6,8 +6,17 @@
     </title>
   </head>
   <body>
-    <h1>Logs</h1>
+    <h2>Logs</h2>
```

Comment:
```suggestion
    <h1>Logs</h1>
```
Importance decreaces by number
H1 - Most
H6 - Least